build/commands/lib/gitPatcher.js

```diff
@@ -149,8 +149,8 @@ module.exports = class GitPatcher {
       const patchInfo = JSON.parse(patchInfoRaw)
       return patchInfo
     } catch (err) {
-      err.message = `Error reading Patch Info file at path "${patchInfoPath}": ${err.message}`
-      throw err
+      console.debug(`Exception while reading ${patchInfoPath}: ${err}`);
```

Comment:
This sounds fatal but it isn't anymore. Also `console.debug` won't do much for node, it is the same as `console.log`. Consider not logging at all here, since it will be given `patchApplyReasons.PATCH_INFO_OUTDATED` which will already be logged to the console as: 
```ts
`The corresponding .${extPatchInfo} file was unreadable or not in the correct schema version of ${patchInfoSchemaVersion}.`,
```

Reply:
It's shouldn't be fatal: it's a real case that we can get on CI when the build is stopped.
`console.debug` was added just for SonarCloud.
Ok, let's remove the logs.